fvm/transactionInvoker.go

```diff
@@ -261,16 +263,12 @@ func (i *TransactionInvoker) deductTransactionFees(env *TransactionEnv, proc *Tr
 	}()
 
 	deductTxFees := DeductTransactionFeesInvocation(env, proc.TraceSpan)
-	_, err = deductTxFees(proc.Transaction.Payer)
+	// Hardcoded inclusion effort (of 1.0). Eventually this will be dynamic.
+	// Execution effort will be connected to computation used.
+	_, err = deductTxFees(proc.Transaction.Payer, 1_0000_0000, computationUsed)
```

Comment:
1_0000_0000 looks a bit odd to me, the use of 4 zeros instead of 3 zeros is not really human readable.